packages/web/src/scss/components/UNSTABLE_PartnerLogo/_UNSTABLE_PartnerLogo.scss

```diff
@@ -0,0 +1,15 @@
+@use 'theme' as theme;
+@use 'tools';
+
+.UNSTABLE_PartnerLogo {
+    width: fit-content;
+    height: fit-content;
+    background: theme.$background-color;
+}
+
+.UNSTABLE_PartnerLogo > img,
+svg {
```

Comment:
```suggestion
.UNSTABLE_PartnerLogo > svg {
```

Reply:
I see there is only `width` set to the image.

When using NextJS and its `Image` component, there are required properties `width` and `height` so when both attributes are rendered in HTML, the CSS must keep its aspect ratio.

Reply:
Thanks for the comments 😉

As @dlouhak mentioned, the height is set  via mixin according to the size of the logo. It is basically the only "static" dimension.

As for NEXTJS, I am aware of this. There is actually one more option how to define sizing of the image - through the `fill` property, but that would still not solve anything for this case. 

I believe this solution should cover all cases

Reply:
Thanks, I missed that 👍